Localize/lang/strings.json

```
@@ -1037,6 +1041,7 @@
   "_AO6T9u.comment": "Done Label for button",
   "_AX7dq3.comment": "Error message for when input node type does not match schema node type",
   "_AYaqW/.comment": "Timezone value ",
+  "_AbZRJ7.comment": "accessability label for a button to select all text in a value box",
```

Why isn't the first letter capitalized?

```suggestion
  "_AbZRJ7.comment": "Accessability label for a button to select all text in a value box",
```